proto/sentry_protos/snuba/v1/endpoint_trace_item_columns.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message TraceItemColumnsRequest {
+  message OrderBy {
+    AttributeKey key = 1;
+    bool descending = 2;
+  }
+
+  RequestMeta meta = 1;
+  TraceItemFilter filter = 2;
+  repeated OrderBy order_by = 3;
+  repeated AttributeKey keys = 4;
```

Comment:
Is this just a table of raw attribute values? Is there a way to have aggregates in the table?

Reply:
Correct and there are no aggregates, although maybe that's an oversight

Reply:
added aggregates